### PR TITLE
Add new image

### PR DESCRIPTION
**Pull request by ocefpaf**
We need to do the same for the prod file and push to the prod branch if this works.

Merging this soon so we can test it.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Failing with:

```
Run docker://sebialvis/hubploy:2020040816492112c8c5
/usr/bin/docker run --name sebialvishubploy2020040816492112c8c5_c3f5d8 --label 87c201 --workdir /github/workspace --rm -e INPUT_ARGS -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/ohw-jupyterhub-2020/ohw-jupyterhub-2020":"/github/workspace" sebialvis/hubploy:2020040816492112c8c5 build hackweek-hub
Traceback (most recent call last):
  File "/usr/local/bin/hubploy", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/hubploy/__main__.py", line 77, in main
    with auth.registry_auth(args.deployment, args.push, args.check_registry):
  File "/usr/local/lib/python3.7/contextlib.py", line 114, in __enter__
    raise RuntimeError("generator didn't yield") from None
RuntimeError: generator didn't yield
```

@salvis2 do we need to update any secrets here? Or I should've pushed directly to the staging branch?

**Comment by salvis2** [comment]
Ah, there's some funny stuff with Hubploy. It's failed on the image building, which we aren't using, so we're calling it a success. :P I'd add the settings to `prod.yaml` in here, merge, and then test on the staging hub.

**Comment by salvis2** [comment]
I'll edit the GitHub Actions to ignore that particular action so this doesn't happen in the future. Will do that after this gets merged.

File: deployments/ohw-hub/config/staging.yaml [hunk 1]
```diff
@@ -9,13 +9,13 @@ jupyterhub:
       hosts:
         - staging.ocean.hackweek.io
       letsencrypt:
-        contactEmail: scottyh@uw.edu
+        contactEmail: mayorga@apl.washington.edu
```
Comment (salvis2) [inline deployments/ohw-hub/config/staging.yaml:12]: This should be something for HTTPS: Scott's email is the one that the hackweek.io domain is registered under. I'll revert this line.

**Comment by salvis2** [comment]
Well, I couldn't figure out how to edit your branch directly, so I made an equivalent commit here: 3cb227e9a4df7505edae75a08581f2ddf9b018c9

I can see the image, launch my server on the staging hub, and open a notebook with an R kernel. This commit  overwrites this PR.

**Comment by ocefpaf** [comment]
> Well, I couldn't figure out how to edit your branch directly, so I made an equivalent commit here: [3cb227e](https://github.com/oceanhackweek/ohw-jupyterhub-2020/commit/3cb227e9a4df7505edae75a08581f2ddf9b018c9)
> 
> I can see the image, launch my server on the staging hub, and open a notebook with an R kernel. This commit overwrites this PR.

I forgot to click on the allow edits by maintainers :-(

No problem. Do what you have to do. I'm tied up today and tomorrow but it looks like we have everything we need to automate the image builds.

PS: I'll be sending a few extra packages soon, so we'll know by then :-)

**Pull request closed without merging**